### DISCUSSION TITLE
Mutate columns that have spaces in names

### DISCUSSION THREAD

**Discussion by jjAtNxtPt** [ibis-project/ibis#10399] [Q&A] [answered]
I have a data frame that has columns containing a space in the name.  I'd like to update those columns inside a mutate but can't seem to figure out how to do it.  I was thinking something like this but I get an error saying the expression cannot contain an assignment:

`.mutate(
    "SOM COL" = _["SOM COL"].fill_null(0) + 200
)`

Is this possible in some way?

**Comment by cpcloud** [comment, marked as answer]
Yes, like this:

```python
.mutate({"SOM COL": _["SOM COL"].fill_null(0) + 200})
.mutate(**{"SOM COL": _["SOM COL"].fill_null(0) + 200})
```

These two lines have equivalent behavior.

**Reply by jjAtNxtPt** [reply]
Thanks!
